server/monitor.go

```diff
@@ -2606,22 +2609,21 @@ func (s *Server) Jsz(opts *JSzOptions) (*JSInfo, error) {
 		opts.Accounts = true
 	}
 
-	// Check if we want a response from the leader only.
-	if opts.LeaderOnly {
```

Comment:
I am not familiar with `/jsz` endpoint, but I now realize that there was already an option to have just the result from the leader, no? So are we making this Jsz option deprecated?

Reply:
That being said, at least from http monitoring, this option would never be set, so always false..

Reply:
Indeed, the thing this tries to improve is that users a) dont know of this setting b) gets confused when non leaders have different information from the leader - only the leader has accurate information.

To avoid the confusion we'll rather not show the information where we know its wrong.

Reply:
And now I wonder if the settings I am setting - leader - is known to be correct always even on non leader? @derekcollison ?

Reply:
Agreed. Few days ago we were doing a Zoom with Derek and were checking the /jsz page and thought there was a problem, until Derek remembered that the info was correct only from the leader.